omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/SessionController1_8Test.java

```diff
@@ -77,4 +96,26 @@ public void get_shouldReturnTheSessionIdIfTheUserIsNotAuthenticated() throws Exc
 		Assert.assertEquals(SESSION_ID, PropertyUtils.getProperty(ret, "sessionId"));
 		Assert.assertEquals(false, PropertyUtils.getProperty(ret, "authenticated"));
 	}
+	
+	@Test
+	public void post_shouldSetTheUserLocale() throws Exception {
+		Locale newLocale = new Locale("sp");
+		String content = "{\"locale\":\"" + newLocale.toString() + "\"}";
+		controller.post(new ObjectMapper().readValue(content, HashMap.class));
+		Assert.assertEquals(newLocale, Context.getLocale());
```

Comment:
Can you add Assert.assertNotEquals(newLocale, Context.getLocale()) before posting?